src/main/java/net/revelc/code/formatter/json/JsonFormatter.java

```diff
@@ -41,12 +43,21 @@ public void init(Map<String, String> options, ConfigurationSource cfg) {
 
         int indent = Integer.parseInt(options.getOrDefault("indent", "4"));
         String lineEnding = options.getOrDefault("lineending", SystemUtil.LINE_SEPARATOR);
+        boolean spaceBeforeSeparator = Boolean.parseBoolean(options.getOrDefault("spaceBeforeSeparator", "true"));
 
         formatter = new ObjectMapper();
 
         // Setup a pretty printer with an indenter (indenter has 4 spaces in this case)
         DefaultPrettyPrinter.Indenter indenter = new DefaultIndenter(Strings.repeat(" ", indent), lineEnding);
-        DefaultPrettyPrinter printer = new DefaultPrettyPrinter();
+        DefaultPrettyPrinter printer = new DefaultPrettyPrinter() {
+            public DefaultPrettyPrinter withSeparators(Separators separators) {
```

Comment:
I don't know much about this formatter. It's not clear to me from the PR how this configuration option is set. Is it set in the Maven configuration? Or, like the Java formatter, in its configuration/style file?

Reply:
yes. this config can be set via maven
```
<groupId>net.revelc.code.formatter</groupId>
<artifactId>formatter-maven-plugin</artifactId>
<configuration>
   <sourceDirectory>${basedir}</sourceDirectory>
   <configJsonFile>json-style.properties</configJsonFile>
</configuration>
```
where `json-style.properties` can be
```
indent=4
spaceBeforeSeparator=false
```